content/version/0.4.301/docs/getstarted/get-started-cluster.md

```diff
@@ -137,7 +139,8 @@ export SYSTEM_MANAGER_URL=<IP ADDRESS OF THE NODE HOSTING THE ROOT ORCHESTRATOR>
 ```
 Then, download setup and startup the root orchestrator simply running:
 ```
-curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/develop/scripts/StartOakestraRoot.sh | sh - 
+export OAKEYSTRA_BRANCH=v0.4.301
+curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/alpha-v0.4.302/scripts/StartOakestraRoot.sh | sh - 
```

Comment:
For compatibility reasons, only the start root and start cluster scripts reference a specific version of the script working for them. More details about why here: https://github.com/oakestra/oakestra/issues/375

The rest of v0.4.301 and all the new ones will use the version agnostic startup proposed here.